stor/dx.py

```diff
@@ -286,11 +509,14 @@ def listdir(self, only='all', canonicalize=False):
                         proj_id, (entry.lstrip('/') if key == 'folders' else entry['id']))))
                 else:
                     if key == 'folders':
-                        ans_list.append(DXVirtualPath(self.drive + proj_name + ':' + entry))
+                        ans_list.append(DXVirtualPath('{drive}{proj_name}:{folder}'.format(
```

Comment:
you should change above to also use format string for `{drive}`, currently it's `dx://{}:/{}`